updatecli/updatecli.d/patch-kubewarden-controller-with-crds-update.yaml

```diff
@@ -0,0 +1,168 @@
+name: Update kubewarden-controller chart patch version with CRDs update
+
+sources:
+  controllerChart:
+    name: Load chart version
+    kind: yaml
+    spec:
+      file: "file://charts/kubewarden-controller/Chart.yaml"
+      key: "version"
+  controllerChartValuesFile:
+    kind: yaml
+    spec:
+      file: "charts/kubewarden-controller/chart-values.yaml"
+      key: "image.tag"
+  controllerValuesFile:
+    kind: yaml
+    spec:
+      file: "charts/kubewarden-controller/values.yaml"
+      key: "image.tag"
+  defaultsChartVersion:
+    kind: yaml
+    transformers:
+      - semverinc: "patch"
+    spec:
+      file: "charts/kubewarden-crds/Chart.yaml"
+      key: "version"
+      #  policyServerSource:
+      #    kind: shell
+      #    spec:
+      #      command: echo policyserver
+      #  admissionPolicySource:
+      #    kind: shell
+      #    spec:
+      #      command: echo admissionpolicies
+      #  clusterAdmissionPolicySource:
+      #    kind: shell
+      #    spec:
+      #      command: echo clusteradmissionpolicies
+
```

Comment:
nitpicking: do we need these commented lines?

Reply:
Sorry. I forgot to remote them. Fixed it.